filter/rastertolabel.c

```diff
@@ -292,6 +297,39 @@ StartPage(ppd_file_t         *ppd,	/* I - PPD file */
 	LastSet    = 0;
         break;
 
+    case ZEBRA_ZPL_DIRECT :
+       /*
+	* Start label...
+	*/
```

Comment:
Incorrect spacing here and further down in the file.

Reply:
This looks accurate to me? The file as it stands is a bastardized mix of spaces and tabs. I've tried to match surroundings as best as I could.